### PR TITLE
refactor: rad to cfg

### PR DESCRIPTION
**Pull request by jpangelle**
### Description

This pull request replaces the use of `RAD` throughout the codebase with `CFG`.

[Staging Link](https://6063ad4787bed20008f55f3d--tinlake-kovan-staging.netlify.app/)

The exceptions are listed below:

- Google storage name
  - [tinlake-ui.yml](https://github.com/centrifuge/apps/blob/main/.github/workflows/tinlake-ui.yml#L85)
  - [.env.dev-example](https://github.com/centrifuge/apps/blob/main/tinlake-ui/.env.dev-example#L19)
  - [.env.prod-example](https://github.com/centrifuge/apps/blob/main/tinlake-ui/.env.prod-example#L19)
- Part of Medium article link
  - [containers/UserRewards/index.tsx](https://github.com/centrifuge/apps/blob/main/tinlake-ui/containers/UserRewards/index.tsx#L334)
- References a GCP bucket name
  - [ducks/userRewards.ts](https://github.com/centrifuge/apps/blob/main/tinlake-ui/ducks/userRewards.ts#L82)
- References to `radClaims` chain method
  - [services/centChain/index.ts](https://github.com/centrifuge/apps/blob/main/tinlake-ui/services/centChain/index.ts#L58)
  - [services/centChain/index.ts](https://github.com/centrifuge/apps/blob/main/tinlake-ui/services/centChain/index.ts#L72)

### CODE REVIEW

File: .github/workflows/tinlake-ui.yml [hunk 1]
```diff
@@ -80,6 +80,6 @@ env:
   NEXT_PUBLIC_IPFS_GATEWAY: "https://cloudflare-ipfs.com/ipfs/"
   NEXT_PUBLIC_INFURA_KEY: ${{ secrets.NEXT_PUBLIC_INFURA_KEY }}
   NEXT_PUBLIC_CENTRIFUGE_CHAIN_URL: "wss://fullnode.centrifuge.io"
-  NEXT_PUBLIC_CLAIM_RAD_CONTRACT_ADDRESS: "0x297237e17F327f8e5C8dEd78b15761A7D513353b"
+  NEXT_PUBLIC_CLAIM_CFG_CONTRACT_ADDRESS: "0x297237e17F327f8e5C8dEd78b15761A7D513353b"
```
Comment (jpangelle) [inline .github/workflows/tinlake-ui.yml:83]: Once merged, would need to update environment variable name in Netlify

File: tinlake-ui/components/PoolsMetrics/index.tsx [hunk 2]
```diff
@@ -134,11 +134,11 @@ const PoolsMetrics: React.FC<Props> = (props: Props) => {
       >
         <Box>
           <Cont style={{ marginTop: '8px' }}>
-            <TokenLogo src={`/static/rad.svg`} />
+            <TokenLogo src={`/static/cfg.svg`} />
```
Comment (jpangelle) [inline tinlake-ui/components/PoolsMetrics/index.tsx:137]: Need to add new CFG icon.

File: tinlake-ui/components/WalletRewards/index.tsx [hunk 3]
```diff
@@ -25,16 +25,16 @@ export const WalletRewards = ({ address }: { address: string }) => {
 
   return (
     <Cont direction="row" pad={{ vertical: '10px', horizontal: '16px' }}>
-      <TokenLogo src="/static/rad-black.svg" />
+      <TokenLogo src="/static/cfg-black.svg" />
```
Comment (jpangelle) [inline tinlake-ui/components/WalletRewards/index.tsx:28]: Need to add new icon here as well.

File: tinlake-ui/services/centChain/index.ts [hunk 4]
```diff
@@ -54,8 +54,8 @@ export class CentChain {
     return await (await this.api()).query.system.account(accountId)
   }
 
-  public async claimedRADRewards(addr: string) {
-    const claimed = await (await this.api()).query.radClaims.accountBalances(centChainAddrToAccountId(addr))
+  public async claimedCFGRewards(addr: string) {
+    const claimed = await (await this.api()).query.cfgClaims.accountBalances(centChainAddrToAccountId(addr))
```
Comment (jpangelle) [inline tinlake-ui/services/centChain/index.ts:58]: Will need to coordinate with @mikiquantum on these changes.

File: tinlake.js/src/abi/index.ts [hunk 5]
```diff
@@ -2,7 +2,7 @@ import { ContractAbis } from '../Tinlake'
 import contractAbiActions from './Actions.abi.json'
 import contractAbiAssessor from './Assessor.abi.json'
 import contractAbiAssessorAdmin from './AssessorAdmin.abi.json'
-import contractAbiClaimRAD from './ClaimRAD.abi.json'
+import contractAbiClaimCFG from './ClaimCFG.abi.json'
```
Comment (jpangelle) [inline tinlake.js/src/abi/index.ts:5]: Is it safe to rename the abi?

File: .github/workflows/tinlake-ui.yml (hunk 1 above)
Comment (jpangelle) [inline .github/workflows/tinlake-ui.yml:83]: Once merged, would need to update environment variable name in Netlify

File: tinlake-ui/components/PoolsMetrics/index.tsx (hunk 2 above)
Comment (jpangelle) [inline tinlake-ui/components/PoolsMetrics/index.tsx:137]: Need to add new CFG icon.

File: tinlake-ui/components/WalletRewards/index.tsx (hunk 3 above)
Comment (jpangelle) [inline tinlake-ui/components/WalletRewards/index.tsx:28]: Need to add new icon here as well.

File: tinlake.js/src/abi/index.ts (hunk 5 above)
Comment (jpangelle) [inline tinlake.js/src/abi/index.ts:5]: Is it safe to rename the abi?

**Comment by philipstanislaus** [comment]
> The exceptions are listed below:
> 
> * Google storage name
>   
>   * [tinlake-ui.yml](https://github.com/centrifuge/apps/blob/main/.github/workflows/tinlake-ui.yml#L85)
>   * [.env.dev-example](https://github.com/centrifuge/apps/blob/main/tinlake-ui/.env.dev-example#L19)
>   * [.env.prod-example](https://github.com/centrifuge/apps/blob/main/tinlake-ui/.env.prod-example#L19)
> * Part of Medium article link
> 
>   * [containers/UserRewards/index.tsx](https://github.com/centrifuge/apps/blob/main/tinlake-ui/containers/UserRewards/index.tsx#L334)
> * References a GCP bucket name
>   
>   * [ducks/userRewards.ts](https://github.com/centrifuge/apps/blob/main/tinlake-ui/ducks/userRewards.ts#L82)
> * References to `radClaims` chain method
>   
>   * [services/centChain/index.ts](https://github.com/centrifuge/apps/blob/main/tinlake-ui/services/centChain/index.ts#L58)
>   * [services/centChain/index.ts](https://github.com/centrifuge/apps/blob/main/tinlake-ui/services/centChain/index.ts#L72)

@mikiquantum could you ping @jpangelle before the bucket names and the function name from the chain is updated on mainnet?

File: .github/workflows/tinlake-ui.yml (hunk 1 above)
Comment (philipstanislaus) [reply .github/workflows/tinlake-ui.yml:83]: Just added that new variable form tinlake.centrifuge.io, so we just need to remove the old `NEXT_PUBLIC_CLAIM_RAD_CONTRACT_ADDRESS` and won't have any downtime.

File: tinlake-ui/containers/UserRewards/index.tsx [hunk 6]
```diff
@@ -325,16 +325,16 @@ const Explainer = () => (
       <HelpIcon src="/static/help-circle.svg" />
       <h3 style={{ margin: 0 }}>How it works</h3>
     </Box>
-    Radial (RAD) Rewards are earned on Ethereum based on your Tinlake investments but claimed on Centrifuge Chain. To
-    claim your rewards you need to link your Tinlake investment account to a Centrifuge Chain account receiving and
-    holding your RAD.
+    Centrifuge (CFG) Rewards are earned on Ethereum based on your Tinlake investments but claimed on Centrifuge Chain.
```
Comment (philipstanislaus) [inline tinlake-ui/containers/UserRewards/index.tsx:328]: This reads a bit weird, can we just say:

```suggestion
    CFG rewards are earned on Ethereum based on your Tinlake investments but claimed on Centrifuge Chain.
```

File: tinlake.js/src/abi/index.ts (hunk 5 above)
Comment (philipstanislaus) [reply tinlake.js/src/abi/index.ts:5]: Yes 👍

**Review by philipstanislaus — APPROVED** [review]
LGTM, thanks 👍

**Pull request merged**